### PR TITLE
[FSDE-2133] Prepared Chi 5.1.0 release

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/FSDE-2133

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-139/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://561c597cce8e540ffa453229b4227234b8dd4501-chi-pr.enterprise.ctl.io

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
 # CHANGELOG
+## 5.1.0 (August 5, 2021)
+### Components
+#### Added
+* Added: Application layout web component and html blueprint templates now supports optional help icon.
```
Comment (janibolkvadze) [inline CHANGELOG.md:5]:
```suggestion
* Added: Application layout web component and HTML blueprint templates now support optional help icon.
```

File: src/website/views/getting-started/whats-new.pug [hunk 2]
```diff
@@ -3,6 +3,33 @@ title: What's new
...
 ---
 
+.changelog__version.-py--1
+  h2.-text--h2.-text--bold.-mb--1 <a href="https://github.com/CenturyLink/Chi/releases/tag/v5.1.0" target="_blank">Chi v5.1.0</a>
+  .-text--grey.-text--md August 5, 2021
+  .-pt--1
+    h3.-text--h3.-text--bold Components
+    h4.-text--h4.-text--bold Added
+    ul.-text.-pl--2
+      li Added: Application layout web component and html blueprint templates now supports optional help icon.
```
Comment (janibolkvadze) [inline src/website/views/getting-started/whats-new.pug:13]:
```suggestion
      li Added: Application layout web component and HTML blueprint templates now support optional help icon.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
+* Added: Data Table Vue component now supports `chiSelectAll` and `chiDeselectAll`.
+#### Changed
+* Changed: Stat component has been improved to support new portal theme designs.
+* Changed: Drawer close icon color has been changed from black to blue for portal themed drawers.
```
Comment (janibolkvadze) [inline CHANGELOG.md:9]:
```suggestion
* Changed: Drawer close icon color has been changed from black to blue for portal-themed drawers.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,22 @@
...
+* Changed: Icons used for decoration or visual styling now include an aria-hidden=“true” attribute for accessibility.
+* Changed: Portal themed data table striping is now an optional style.
```
Comment (janibolkvadze) [inline CHANGELOG.md:11]:
```suggestion
* Changed: Portal-themed data table striping is now an optional style. 
```

File: src/website/views/getting-started/whats-new.pug [hunk 5]
```diff
@@ -3,6 +3,33 @@ title: What's new
...
+      li Added: Data Table Vue component now supports <code>chiSelectAll</code> and <code>chiDeselectAll</code>.
+    h4.-text--h4.-text--bold Changed
+    ul.-text.-pl--2
+      li Changed: Stat component has been improved to support new portal theme designs.
+      li Changed: Drawer close icon color has been changed from black to blue for portal themed drawers.
```
Comment (janibolkvadze) [inline src/website/views/getting-started/whats-new.pug:18]:
```suggestion
      li Changed: Drawer close icon color has been changed from black to blue for portal-themed drawers.
```

File: src/website/views/getting-started/whats-new.pug [hunk 6]
```diff
@@ -3,6 +3,33 @@ title: What's new
...
+      li Changed: Icons used for decoration or visual styling now include an aria-hidden=“true” attribute for accessibility.
+      li Changed: Portal themed data table striping is now an optional style.
```
Comment (janibolkvadze) [inline src/website/views/getting-started/whats-new.pug:20]:
```suggestion
      li Changed: Portal-themed data table striping is now an optional style.
```

File: src/website/views/getting-started/whats-new.pug [hunk 7]
```diff
@@ -3,6 +3,33 @@ title: What's new
...
+    h4.-text--h4.-text--bold Fixed
+    ul.-text.-pl--2
+      li Fixed: Text utility bug which caused margin on text headings to take precedence over margin utility classes.
+      li Fixed: Dropdown bug which caused dropdown menu item text to inherit the color and font-weight of its parent container.
+      li Fixed: Icon webfont was returning an ‘Unexpected missing generic font family’ bug in sonar.
```
Comment (janibolkvadze) [inline src/website/views/getting-started/whats-new.pug:25]:
```suggestion
      li Fixed: Icon webfont was returning an <code>Unexpected missing generic font family</code> bug in sonar.
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Review by dani-cl-madrid — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-139/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://69c7bdea709e1a8856f96c6e55d2df36f04866d6-chi-pr.enterprise.ctl.io

**Pull request merged**